ci/pipeline-production.yml

```diff
@@ -550,15 +567,26 @@ resource_types:
   - name: github-release
     type: registry-image
     source:
-      repository: concourse/github-release-resource
+      aws_access_key_id: ((ecr_aws_key))
+      aws_secret_access_key: ((ecr_aws_secret))
+      repository: github-pr-resource
```

Comment:
This is hardened resource type is not the same as `concourse/github-release-resource`. We can see if we can add it to common pipelines.

Reply:
My mistake. Would the common-piplines git-resource cover what we need?

Reply:
No, there is a specific release resource on the list though, I can't remember if it has been hardened yet

Reply:
https://github.com/cloud-gov/product/issues/2618, not hardened yet but tracked, we should use the existing one for now

Reply:
@drewbo Isn't that the one I'm using? That Andrew said isn't right here?

Reply:
we should use the unhardened `concourse/github-release-resource` here (so no change from the original file). It is scheduled to be hardened (linked ticket) and we can replace it then

Reply:
Got it. Thanks.